lib/ui/alert_detail.dart

```diff
@@ -339,7 +339,9 @@ class _AlertDetailPageState extends State<AlertDetailPage> {
                     ].expand(
                           (widget) => [
                         widget,
-                        const SizedBox(
+                        (widget.runtimeType == Offstage && (widget as Offstage).offstage)
+                            ? const SizedBox.shrink()
```

Comment:
what if we do something like:

                          (widget) => (widget.runtimeType == Offstage && (widget as Offstage).offstage) ?
                            [widget] : [widget, const SizedBox(height: 24,)]

Reply:
> what if we do something like:
> 
> ```
>                       (widget) => (widget.runtimeType == Offstage && (widget as Offstage).offstage) ?
>                         [widget] : [widget, const SizedBox(height: 24,)]
> ```

i think that should work too! improves readability as well haha